app/src/main/java/com/teamagam/gimelgimel/app/view/viewer/cesium/CesiumMapView.java

```diff
@@ -264,6 +270,15 @@ public boolean isReady() {
         return mIsGGMapReadySynchronized.getData();
     }
 
+    @Override
+    public void onError(String error) {
+        if(!mIsHandlingError.getData()) {
+            mIsHandlingError.setData(true);
+            init();
+            loadUrl(FILE_ANDROID_ASSET_VIEWER);
```

Comment:
why do you load the url second time?
you have the same line at the end of init !fix